src/dune_file.ml

```diff
@@ -1948,6 +1948,9 @@ type Stanza.t +=
 module Stanzas = struct
   type t = Stanza.t list
 
+  let hash = Hashtbl.hash
+  let equal = (=)
```

Comment:
This seems to be somewhat incorrect because at least some stanzas have some Sets in them.
I can't see where it's used either. Do we need it?

Reply:
I'll get rid of it. It's from my first attempt at using a different input.